### PR TITLE
watch-termination: don't log oauth tokens

### PR DESCRIPTION
**Pull request by sttts**
We log requests of oauth tokens when log level is increased. But in the default case we don't log any requests. This PR restore the old (sub-optimal) situation, i.e. we are not worse than before.

### CODE REVIEW

File: cmd/watch-termination/main.go [hunk 1]
```diff
@@ -123,7 +139,7 @@ func (w *terminationFileWriter) Write(bs []byte) (int, error) {
 				Compress:   false,
 			}
 			w.logger = l
-			klog.Infof("Redirecting termination logs to %q", w.fn)
+			fmt.Fprintf(os.Stderr, "Copying termination logs to %q\n", w.fn)
```
Comment (sttts) [inline cmd/watch-termination/main.go:142]: with the recursive `klog.Infof` there was no logging at all.

**Comment by p0lyn0mial** [comment]
requires https://github.com/openshift/origin/pull/25122

**Comment by sttts** [comment]
Infra.

/retest

**Pull request merged**